TestingApp/Settings/SettingsViewController.swift

```diff
@@ -199,79 +249,54 @@ private extension SettingsViewController {
         sections[0] = configurationSection
     }
 
-    private func loadConf() -> Configuration {
-        let siteApiKeyId = UserDefaults.standard.string(forKey: "conf.siteApiKeyId") ?? ""
-        let siteApiKeySecret = UserDefaults.standard.string(forKey: "conf.siteApiKeySecret") ?? ""
-        let site = UserDefaults.standard.string(forKey: "conf.site") ?? ""
-        let visitorAssetId = loadVisitorContextAssetId()
-        let visitorContext = UUID(uuidString: visitorAssetId)
-            .map(Configuration.VisitorContext.init(assetId:))
-        return Configuration(
-            authorizationMethod: .siteApiKey(
-                id: siteApiKeyId,
-                secret: siteApiKeySecret
-            ),
-            environment: .beta,
-            site: site,
-            visitorContext: visitorContext
+    private func updateConfiguration() {
+        let uuid = UUID(uuidString: visitorContextAssedIdCell.textField.text ?? "")
+        props.changeConfig(
+            .init(
+                authorizationMethod: .siteApiKey(id: siteApiKeyIdCell.textField.text ?? "", secret: siteApiKeySecretCell.textField.text ?? ""),
```

Comment:
I think that here it would be beneficial to have the authorization method be created in its own variable where you can format it nicely, and then use it here. Would avoid the line being so long

Reply:
fixed - 1dd9a6d247757d77b9f69974e2e98c719d91741e